### PR TITLE
feat: show the source of processes/ingredients/materials in the explorer

### PR DESCRIPTION
**Pull request by ccomb**
[story](https://www.notion.so/4f0841edbc29408eb147558e53fe2acd?v=40d0d544eb8e4c03891da1f9aad8b6cd&p=d8a198cc796b4d3ead88c3a055043f26&pm=s)

### CODE REVIEW

**Review by n1k0 — COMMENTED** [review]
Code looks fine. I'll let @paulboosz validate and approve the data updates.

**Review by paulboosz — APPROVED** [review]

File: public/data/food/processes_impacts.json [hunk 1]
```diff
@@ -327,40 +336,42 @@
...
-      "ozd": 4.733772908e-7,
-      "pco": 0.03956660902,
-      "pma": 0.000002644178814,
-      "swe": 0.06590072544,
-      "tre": 1.766745681,
-      "wtu": 2.987814735,
-      "ecs": 2186.0104454838856,
-      "pef": 2532.5378885785976
+      "acd": 0.3827678496,
+      "cch": 25.03890328,
```
[the remark this answers is not in the document]
Comment (paulboosz) [reply public/data/food/processes_impacts.json:347]: I just re-ran `make export_food` on this branch on the linux vm and I have a diff in impacts... Definitely a brightway bug here

image

[the remark this answers is not in the document]
Comment (ccomb) [reply public/data/food/processes_impacts.json:347]: I'll relaunch the whole process after a `make clean` and `docker system prune`. I persistently have differences like that, specifically on the ground beef. This is the only process on which it happens. Is there a way to definitely know which value is correct?

[the remark this answers is not in the document]
Comment (ccomb) [reply public/data/food/processes_impacts.json:347]: I confirm I still get the same result after a full run from zero

[the remark this answers is not in the document]
Comment (ccomb) [reply public/data/food/processes_impacts.json:347]: Running again a `make export_food` gives the same results

[the remark this answers is not in the document]
Comment (ccomb) [reply public/data/food/processes_impacts.json:347]: Running yet again a make `export_food` gives again the same results

**Comment by ccomb** [comment]
I've reverted the unwanted impact changes

**Pull request merged**